src/compiler/scala/tools/nsc/backend/jvm/BCodeHelpers.scala

```diff
@@ -882,8 +882,8 @@ abstract class BCodeHelpers extends BCodeIdiomatic with BytecodeWriters {
       var fieldList = List[String]()
 
       for (f <- fieldSymbols if f.hasGetter;
-	         g = f.getter(cls);
-	         s = f.setter(cls);
+	         g = f.getterIn(cls);
+	         s = f.getterIn(cls);
```

Comment:
Both in this changeset and in the new, squashed one there are a lot of places, where you changed setter to getterIn? Why? Was it intentional?